src/main/resources/assets/js/app/security/FilterablePrincipalLoader.ts

```diff
@@ -0,0 +1,56 @@
+import {PrincipalLoader} from 'lib-admin-ui/security/PrincipalLoader';
+import {FindPrincipalListRequest} from 'lib-admin-ui/security/FindPrincipalListRequest';
+import {Principal} from 'lib-admin-ui/security/Principal';
+import {PrincipalKey} from 'lib-admin-ui/security/PrincipalKey';
+
+export class FilterablePrincipalLoader extends PrincipalLoader {
+
+    private static PROJECT_ROLE_PREFIX: string = 'cms.project.';
+
+    private forbiddenPrincipals: { [key: string]: PrincipalKey; } = {};
+
+    constructor() {
+        super();
+    }
+
+    resetForbiddenPrincipals(): FilterablePrincipalLoader {
+        this.forbiddenPrincipals = {};
+
+        return this;
+    }
+
+    skipPrincipals(principalKeys: PrincipalKey[]): FilterablePrincipalLoader {
+        principalKeys.forEach((principalKey: PrincipalKey) => {
+            this.forbiddenPrincipals[principalKey.toString()] = principalKey;
+        });
```

Comment:
change this to 

`principalKeys.forEach((principalKey: PrincipalKey) => this.skipPrincipal(principalKey)); `